---
layout: documentation
lang-ref: beanbag/70/issue-70-phylogenomics-working-group
permalink: /beanbag/70/issue-70-phylogenomics-working-group
title: Phylogenomics Working Group
description: Phylogenomics Working Group
sideNavigation: sidenav.beanbag70
---

# PHYLOGENOMICS WORKING GROUP

Coordinators:  
**Félix Forest** (Royal Botanic Gardens, Kew, UK)  
**Rafaela Trad** (Royal Botanic Gardens, Edinburgh, UK)  

Here is a round-up of mini-reports on legume phylogenomics projects contributed by Bean Bag readers.

#### Phylogenomics of legumes with a focus on subfamily Papilionoideae

**Rafaela Trad** (<rtrad@rbge.org.uk>)  

Over the last year, we have made much progress towards generating a complete genus-level phylogeny of the legumes with a particular focus on Papilionoids based on target enrichment sequencing using the universal probe set Angiosperms353. This work is part of the Plant and Fungal Trees of Life project at RBG Kew (<https://www.kew.org/science/our-science/projects/plant-tree-of-life>), which usually samples only one species/specimen per genus. Currently, we have over 95% of the currently recognised genera sequenced and analyses are ongoing. We aim to use the resulting phylogenetic tree to review the classification of subfamily Papilionoideae. We are particularly keen for the phylogeny to be a useful tool for the legume research community to address questions across the family and subfamily Papilionoideae.  

We still need material for the genus *Neocolletia*. Material for all the other genera will be processed in the near future, or has already been sequenced. If you have access to tissue or DNA for this genus, please get in touch with Félix Forest (<f.forest@kew.org>) or Rafaela Trad (<rtrad@rbge.org.uk>).  

**Team**: Rafaela Trad, Greg Kenicer and Flávia Pezzini (Royal Botanic Gardens Edinburgh, UK), Toby Pennington (Royal Botanic Gardens Edinburgh, UK and University of Exeter, UK), Félix Forest, Gwilym Lewis and Bente Klitgaard (Royal Botanic Gardens, Kew, UK).  

#### Legume fossil database

**Jia Linbo** (<jialinbo@mail.kib.ac.cn>)  

Over the past thirty years, the Paleobotanical Research Group at the Kunming Institute of Botany (Chinese Academy of Sciences) has collected approximately 800 well-preserved legume fossils from the Paleocene to Pliocene of East Asia. We are currently working towards the publications of these fossils and have to date published eight papers, which include the first fossil records of *Pterolobium* and *Mucuna*, as well as the earliest fossil records of *Bauhinia* and the oldest fossil records of *Cladrastis* in Asia. Additionally, we are preparing to report our findings from genera *Zenia*, *Mezoneuron*, *Peltophorum*, and *Christia*, with many of these fossils being the earliest discovered for these groups. Alongside these fascinating discoveries, we aim to investigate the evolution of Leguminosae using fossil evidence and to test hypotheses through molecular studies.  

#### Rapid radiation in *Inga*

**Rowan Schley** (<rowan.schley@gmail.com>)  

We are currently examining whether hybridisation catalysed the rapid radiation of *Inga*, a ubiquitous genus of rainforest trees from tropical America. Together, the *Inga* team have sequenced 1320 target capture loci for >1000 accessions, comprising >200 of ca. 300 *Inga* species. In addition to this, we have produced population-level, whole-genome resequencing data for >530 *Inga*s, comprising 40 species, as well as three chromosomally-contiguous, annotated *Inga* reference genomes with the Sanger Institute. We will use this genomic dataset to assess whether hybridisation transfers chemical defence gene 'cassettes' between *Inga* species, given that chemical defences are key in the high levels of *Inga* species coexistence. Finally, we recently submitted our first paper from the project examining the phylogenomics of hybridisation and introgression in *Inga*, which is currently in review.  

**Team**: Rowan Schley (University of Exeter, UK), Toby Pennington (University of Exeter & RBG Edinburgh, UK), Alex Twyford, Kyle Dexter, Catherine Kidner & Kelly Bocanegra-González (University of Edinburgh & RBG Edinburgh, UK), Rosalia Piñeiro (University A Coruña, Spain), Flávia  Pezzini (RBG Edinburgh, UK), María-José Endara (UDLA, Ecuador), Dale Forrister (STRI, Panama), James  Nicholls (CSIRO, Australia), Jens Ringelberg (University of Edinburgh, UK) and Colin Hughes (University of Zurich, Switzerland).  

![](/assets/images/70/Inga_setosa_RowanSchley_sq.jpg)
*Inga setosa* at Los Amigos Biological Station, Peru. Photo by Rowan Schley.

#### Phylogenomics of the Sindora clade

**Le Min Choo** (<lc988@exeter.ac.uk>)  

The Sindora clade consists of the five genera, *Sindora*, *Sindoropsis*, *Tessmannia*, *Copaifera* and *Detarium*. The aim of this study is firstly to investigate generic limits, especially as *Copaifera* is known to be non-monophyletic, and the disjunct distributions of *Sindora* (1 species in Africa, 19 in Asia) and *Copaifera* (37 in the Neotropics, 4 in Africa, 1 in Asia). Secondly, we aim to look for evidence of past introgression within the clade that might explain morphological similarities among the genera (e.g., spiky pods in *Sindora* and *Tessmannia*, brightly coloured arils in *Sindora* and *Copaifera*). This study is using the genes recovered using the Detarioideae-specific target-capture bait set. Recent samples (silica dried or herbarium) of neotropical *Copaifera*, *Tessmannia* spp. and *Sindoropsis letestui* will be very much appreciated!  

![](/assets/images/70/Sindora-Clade.png)


#### Phylogenomics of *Lupinus* (Papilionoideae) 

**Bruno Nevado** (<bnevado@ciencias.ulisboa.pt>)  

Sequencing large numbers of whole genomes provides new opportunities to gain insights into the genomic and phenotypic bases of evolutionary diversification and to build next-generation phylogenies. We are doing this for the western New World Lupinus clade which comprises > 200 species and includes a series of recent, rapid, nested and parallel evolutionary radiations in western North America and the Andes. So far, a high-quality, chromosome-level reference genome for the Andean crop lupin, *Lupinus mutabilis* has been assembled and 282 whole genomes sequenced at lower coverage for species across this clade.  

**Team**: Bruno Nevado (University of Lisbon, Portugal), Romulo Segovia (University of British Columbia, Canada) & Colin Hughes (University of Zurich, Switzerland)  

![](/assets/images/70/Lupinus_plate.jpg)
Andean species of *Lupinus*, left to right: *L. luisanae*, 3800 m elevation, Cordillera Oriental, Colombian Andes, photo by Natalia Contreras and *L. ananeanus*, 4750 m elevation, Cordillera Real, Bolivian Andes, photo by Colin Hughes.  

#### *Pterocarpus* and allied genera 

**Laura Albreht** (<laura.albreht@ulb.be>) & **Bente Klitgaard** (<b.klitgaard@kew.org>)  

*Pterocarpus* and allied genera are the focus of research led by Bente Klitgaard at the Royal Botanic Gardens, Kew. To-date phylogenetic and phylogenomic research supported by morphometric analyses by the Kew-led group has resulted in the first phylogeny of *Pterocarpus* (Saslis-Lagoudakis *et al.* 2011; doi.org/10.1371/journal.pone.0022275) and biogeographical analyses of the *Pterocarpus* clade (Schley *et al.* 2022; doi.org/10.1111/jbi.14310). Furthermore, we have resolved the Neotropical *Pterocarpus rohrii* species complex, and in the process circumscribed 10 species, including one species new to science (Klitgaard *et al.* in preparation). Our current foci are on resolving the inter- and infra-generic relationships in the Pterocarpus clade and on producing a full taxonomic revision of Neotropical *Pterocarpus*.  

Laura Albreht's doctoral project (Université Libre de Bruxelles, Belgium) tackles species delimitation of CITES-listed African *Pterocarpus* (Papilionoideae), using a phylogenomic approach in addition to morphological and ecological data. As a starting point, a large phylogeny including multiple samples of all accepted species has been reconstructed using the Angiosperms353 gene set, helping to identify taxa for further investigation of potential cryptic diversity. Additionally, we explored the potential of high-copy DNA for identifying African *Pterocarpus* species, by reconstructing and comparing chloroplast and ribosomal DNA phylogenies. The results will be useful for the development of tools facilitating the enforcement of CITES regulations. Laura's PhD is carried out in collaboration with CITES, Herbarium of Royal Botanic Gardens, Kew, Herbarium of Meise Botanic Garden and Royal Museum for Central Africa.  

![](/assets/images/70/Pterocarpus_plate.jpg)
Left to right: Flowers of *Pterocarpus rotundifolius* and fruits of *P. angolensis.* Photos: Gwilym Lewis.  

#### HYPERPLANT: Hyperdominance in plant communities

**Manuel de la Estrella** (<mdelaestrella@uco.es>)  

This new project is funded for the next 4 years (2023-2027) by the Spanish science program and led by M. de la Estrella and L. Pokorny (with collaborators from other institutions/countries). The project will try to investigate the underlying causes of plant hyperdominance using Mediterranean Genisteae as a study group. The main expected HYPERPLANT deliverables are: (1) a comprehensive Genisteae phylogeny, with special focus on the species complex resulting in subtribe Genistinae; (2) establishment of the biogeographic origin of subtribe Genistinae, which will contribute to our understanding of the importance of the Mediterranean biome and the evolution of the xeric vegetation that largely characterizes it; and (3) characterization of the abundance and population genetic diversity patterns currently observed in hyperdominant vs. restricted taxa.

#### BERLiNiA project. Evolutionary ecology of tropical trees in the Berlinia clade

**Dario Isidro Ojeda Alayon** (<dario.alayon@nibio.no>)  

This project targets the Berlinia clade (Detarioideae), an endemic African lineage of tropical trees, with around 16 genera and ca. 180 species. The group is distributed in two main habitat types: forest and savanna biomes. The project aims to investigate the evolutionary history of these two main bioclimatic niches, the drought tolerance/avoidance traits found in the group, and putative syngameons. Duration: 2023-2026.  

**Team**: Dario Isidro Ojeda Alayon (Norwegian Institute of Bioeconomy Research, Norway), Olivier Hardy (Université Libre de Bruxelles, Belgium).  

#### NEXTRAD project. Implementing cost-effective genomic tools for high resolution species delimitation in recent plant island radiations in Macaronesia

**Dario Isidro Ojeda Alayon** (<dario.alayon@nibio.no>)  

This project will use the Angiosperms353 bait set to infer phylogenetic relationships within species-rich, recent island radiations. The main goal of the project is to establish the resolution limit of this bait set when dealing with closely related species of *Lotus* which radiated recently. Duration: 2024-2025.  

**Team**: Dario Isidro Ojeda Alayon (Norwegian Institute of Bioeconomy Research, Norway), Javier Fuertes-Aguilar (Real Jardín Botánico, Madrid, Spain), Ruth Jaen-Molina, (Jardín Botánico Viera y Clavijo, Spain).  

#### Different aspects of the Mediterranean legumes

**Ozan Şentürk** (<osenturk@rbge.org.uk>)  

Based at Royal Botanıc Garden Edinburgh as a PostDoc, I am working on several aspects of Mediterranean Legumes. Firstly, The Illustrated Flora of Türkiye project---co-editing and accounts for the Legume Volumes of this complete and updated revision of the Turkish Flora. Additionally, I am investigating the phylogeny of Mediterranean Genistoids and, as part of a TÜBİTAK grant, the complete phylogeny and biogeography of *Ebenus* L. (Hedysareae). This research will feed into wider aspirations to investigate the Biogeography of the Mediterranean Basin and its surrounds using legumes as a key model group.  

![](/assets/images/70/Mediterranean_plate.jpg)
*Ebenus laguroides* and *Gonocytisus dirmilensis*, photos by Ozan Şentürk.  

#### Phylogenomics and origins of nodulation

**Ryan Folk** (<rfolk@biology.msstate.edu>)  

Many in the legume community will be aware of the recent "NitFix" sequencing effort, sequencing 100 phylogenomic loci to investigate the origins of nodulation. The capstone paper, Kates et al., was recently issued as a preprint and will hopefully follow in print early in the coming year. In this work, evolutionary rate analyses are used to demonstrate phylogenetic evidence for both multiple origins of root nodule symbiosis (RNS) and the elevated lability of symbiotic states in the nitrogen-fixing clade (NFC) ancestor (the "predisposition hypothesis"), overcoming earlier study limitations. An added complexity of the new study was the identification of more complex evolutionary rate variation in RNS than observed previously, doubtlessly due to the greatly improved taxon sampling and resolution of relationships. Finally, key sister nodulating/non-nodulating clades are resolved for the first time, providing an important focus for more detailed genomic and experimental study targeting the reconstruction of what early symbiosis may have looked like. Numerous collaborations have arisen within the NitFix project proceeding from more detailed uses of the data, and members of the legume community will note that this large sequence data release is finalized on the Sequence Read Archive (SRA) as an important resource for future studies.  

Kates H.R., O'Meara B.C., LaFrance R., Stull G.W., James E.K., Conde D., Liu S., Tian Q., Yi T., Kirst M., Ané J.-M., Soltis D.E., Guralnick R.P., Soltis P.S., Folk R.A. 2022. Two shifts in evolutionary lability underlie independent gains and losses of root-nodule symbiosis in a single clade of plants. bioRxiv.:2022.07.31.502231.

#### A phylogenomic survey of *Astragalus* 

**Ryan Folk** (<rfolk@biology.msstate.edu>)  

One of the exciting collaborations resulting from the NitFix initiative focuses on *Astragalus*, the familiar poster-child for mega-genera and an important floristic element in many cold, arid areas of the northern hemisphere. Consistent with the inherent challenges of large, poorly understood clades, *Astragalus* has seen limited prior study from a phylogenomic point of view, primarily using plastid genomes and geographically focused sampling, but a globally scoped sequencing initiative has yielded a strong framework for further study. A pair of preprints advances our understanding of *Astragalus* by exploring (1) diversification, biogeography, and niche evolution and (2) using quantitative methods to investigate phytogeographic patterns in this important north temperate radiation. In both papers the story revolves around Central Asian origins and the importance of edaphic specialization in diversity hotspots (no surprise for nodulating plants!). The study greatly improves upon prior work in terms of taxon and gene sampling, but still has limited taxon coverage of some groups of Eurasian *Astragalus*. Neo-Astragalus also requires further study with stronger sampling but more limited phylogenetic resolution, where evidence of complex paralogy and discord, largely missing in the eastern hemisphere species, indicate the presence of complex evolutionary processes.  

Folk R.A., Charboneau J., Belitz M., Singh T., Kates H.R., Soltis D.E., Soltis P.S., Guralnick R.P., Siniscalchi C.M. 2023a. Anatomy of a mega-radiation: Biogeography and niche evolution in *Astragalus*. bioRxiv.:2023.06.27.546767.  

Folk R.A., Maassoumi A.A., Siniscalchi C.M., Kates H.R., Soltis D.E., Soltis P.S., Belitz M.B., Guralnick R.P. 2023b. Phylogenetic diversity and regionalization in the temperate arid zone. bioRxiv.:2023.11.01.565216.  

#### Phylogenomics of *Mimosa*

**Leonardo Borges** (<aquitemcaqui@gmail.com>)  

Aiming to understand the patterns and processes of morphological disparification in *Mimosa*, we will improve the current phylogenetic hypothesis for the genus with hybrid capture data. In order to increase taxon and loci sampling, we will use sequences already generated for the NitFix project and sequence new samples using both NitFix and Mimobaits probes. Sequencing is planned to start by May 2024 and we are aiming to cover at least half of the 600+ species in *Mimosa*.  

**Team**: Leonardo Borges (Universidade Federal de São Carlos, Brazil), Rafael Barduzzi (Universidade Federal de São Paulo, Brazil), Carolina Siniscalchi (University of Mississippi, USA), Jens Ringelberg (University of Edinburgh, UK), Marcelo Simon (Embrapa Cenargen, Brazil), Ryan Folk (University of Mississippi, USA)  
